tfx/examples/cifar10/setup.py

```diff
@@ -0,0 +1,70 @@
+# Lint as: python2, python3
```

Comment:
just curious, why a new setup is needed? any new dependency for cifar10?

Reply:
We'll need tflite-support and flatbuffers to write TFLite metadata to the model. The setup script was suggested by @zhitaoli in his previous review.